storage/spacesstateless/main.go

```diff
@@ -129,7 +129,7 @@ func (p *Provider) Exists(alias string) (bool, error) {
 }
 
 // Store creates a new short URL
-func (p *Provider) Store(url, alias string) error {
+func (p *Provider) Store(url, alias string, overwrite string) error {
 	exists, err := p.Exists(alias)
```

Comment:
Theres an `exists` case in this block to add your override check to, several of the other providers have them as well so check those for quick code changes to get the other providers working. (don't worry about testing each one)

Reply:
I fixed this for all the providers